src/LibraryManager.Vsix/Shared/VsHelpers.cs

```diff
@@ -285,7 +285,7 @@ public static async Task<bool> DeleteFilesFromProjectAsync(Project project, IEnu
 
                 return true;
             }
-            catch(Exception ex)
+            catch (Exception ex)
```

Comment:
These changes are irrelevant to the PR, can you revert this file?  Formatting will get fixed up the next time we have reason to touch this file.

Reply:
Reverted the formatting changes.